client/components/Canvas/Canvas.js

```diff
@@ -5,11 +5,108 @@ import Diagram from './Diagram/Diagram';
 import Script from './Script/Script';
 
 const Canvas = ({ activeTab }) => {
-  console.log(activeTab);
+  const DUMMY = [
+    {
+      tableName: 'people',
+      columns: [
+        {
+          _id: {
+            primaryKey: true,
+            type: 'SERIAL',
+          },
+        },
+        {
+          name: 'VARCHAR(255)',
+        },
+        {
```

Comment:
Can we please move the dummy data to a separate file and the import from there? That way it makes maintenance and future iteration so much better